doc/tutorials/test-templates-automated.md

```diff
@@ -0,0 +1,179 @@
+
+To build the init container, run the following command:
+
+
+```bash
+gcloud builds submit --config images/publisher/cloudbuild.yaml
+```
+
+Then, to deploy the metrics handler, run these commands:
+
+```bash
+# Deploy the Cloud Function and create the PubSub topic.
+gcloud functions deploy metrics_handler --runtime python37 --trigger-topic=begin-metrics-handler --memory=1024MB --entry-point=run_main --timeout=500s
+# Trigger the Metrics Handler every 15 minutes.
+gcloud scheduler jobs create pubsub metrics-handler --schedule="*/15 * * * *" --topic=begin-metrics-handler --message-body="{}" --description="Kicks off the metric handler"
+```
+
+Note that you can adjust the timing of the metrics handler by changing the `--schedule` argument in the second command above. This flag takes a schedule in [Cron format](https://en.wikipedia.org/wiki/Cron#CRON_expression). You can find the details of the new function on the [Cloud Functions](https://console.cloud.google.com/functions/list) page.
+
+In `mnist-cpu.jsonnet`, add the `schedule` and `publisherImage` fields to the test definition. `schedule` is also in cron format, and `publisherImage` should be the `gcr.io` tag of the image you just built. Finally, replace `mnist.oneshotJob` with `mnist.cronJob`:
+
+```jsonnet
+local mnist = base.BaseTest {
+  ...
+  schedule: '0 */1 * * *',
+  publisherImage: 'gcr.io/your-project/publisher:latest',
+  ...
+};
+
+mnist.cronJob
+```
+
+This will run the MNIST cpu training job at the top of every hour. Build the template with JSonnet:
+
+```bash
+jsonnet -J ml-testing-accelerators/ mnist-cpu.jsonnet
+```
+
+Note the following changes to the output:
+
+1. The `Job` template is now embedded in a `CronJob` resource.
+1. The `Pod` template now has an `initContainer`, `publisher`, that publishes a message to the PubSub queue.
+1. The `METRIC_CONFIG` environment variable of `publisher` automatically tracks the `total_wall_time` of the Job and asserts that it should not deviate by more than 5 standard deviations from the mean.
+
+This `CronJob` will automatically run and its metrics will be periodically collected by the Metrics Handler. Note that, by default, all TensorBoard summaries are collected by the Metrics Handler, but only cerain metrics have assertions.
+
+## Configuring metric aggregation
+
+You can configure metric aggregation and regression alerting with the `metricCollectionConfig` and `regressionTestConfig` field of `BaseTest`, respectively. For this example, we can set a target final accuracy of 98% for the test:
+
+```
+local mnist = base.BaseTest {
+  ...
+  metricCollectionConfig+: {
```

Comment:
add `"default_aggregation_strategies": ["final"],`

Reply:
This is inherited from the base template